Update sidebar.markdown (#19132)
home-assistant/home-assistant.github.io: source/_lovelace/sidebar.markdown

This view doesn't have support for badges.

To change a view to edit mode, or to change the location of a card, enable edit mode:

> Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Then, select your [view (Lovelace tab)](https://www.home-assistant.io/lovelace/views/) and click the edit icon next to its name. At the bottom of the view configuation screen, select the "View type" called sidebar.  

Reviewer: This should not be a part of this page, it should be in <https://www.home-assistant.io/lovelace/views/> that would cover all 3 types and how to change between them.

This page can link to that, but the documentation for that belongs under <https://www.home-assistant.io/lovelace/views/>

Author: I see what you mean @ludeeus, but that's pretty confusing for the user to dig through multiple pages when all they want to know is how to enable sidebar. It's odd that the sidebar page doesn't explain how to enable the feature.

Reviewer: I think this page should have it but in form of a link that goes to the main view page.
Changing the mode is a part of the view configuration, it would be more confusing if it's not a part of that.

Author: I think that forces the user into an unnecessary click. What if we explain how to enable and add "for more info, see the views page" and link there? 

Sidebar is a new feature and I foresee many people being confused and simply looking for how to enable. The views page is pretty busy and would require the user to look through quite a bit.

Reviewer 2: There is a difference between changing an existing view to a sidebar view, or creating a new one. The creating should be on this page, changing an existing view should be on <home-assistant.io/lovelace/views> IMHO

Reviewer 2: But it should def not be in the part that is about editing cards in your sidebar view, as at that point you should already have a sidebar view.

Author: That makes sense to me. What about...

```suggestion
Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Create a new [view (Lovelace tab)](https://www.home-assistant.io/lovelace/views/) and at the bottom of the view configuration screen select the view type called sidebar.  
```
